frontend/ios/DroidKaigi 2019/data/Floor.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Floor.swift
+//  DroidKaigi 2019
+//
+//  Created by 417.72KI on 2019/01/31.
+//
+
+import Foundation
+import class UIKit.UIImage
+
+struct Floor {
```

Comment:
There is only `Floor1`, `Floor2`.
so I think it is better to use `enum`.

Reply:
In any case,I think we need to fix codes when floors changed.🤔

Reply:
Surely... 🤔
I fixed it!